src/render/decorations/CHyprGroupBarDecoration.cpp

```diff
@@ -96,24 +103,31 @@ void CHyprGroupBarDecoration::draw(CMonitor* pMonitor, float a) {
     static auto PTITLEFONTSIZE = CConfigValue<Hyprlang::INT>("group:groupbar:font_size");
     static auto PHEIGHT        = CConfigValue<Hyprlang::INT>("group:groupbar:height");
     static auto PGRADIENTS     = CConfigValue<Hyprlang::INT>("group:groupbar:gradients");
+    static auto PSTACKED       = CConfigValue<Hyprlang::INT>("group:groupbar:stacked");
 
     if (!*PENABLED || !m_pWindow->m_sSpecialRenderData.decorate)
         return;
 
     const auto ASSIGNEDBOX = assignedBoxGlobal();
 
-    m_fBarWidth = (ASSIGNEDBOX.w - BAR_HORIZONTAL_PADDING * (barsToDraw - 1)) / barsToDraw;
+    const auto ONEBARHEIGHT = BAR_PADDING_OUTER_VERT + BAR_INDICATOR_HEIGHT + (*PGRADIENTS || *PRENDERTITLES ? *PHEIGHT : 0);
+    m_fBarWidth             = *PSTACKED ? ASSIGNEDBOX.w - BAR_HORIZONTAL_PADDING : (ASSIGNEDBOX.w - BAR_HORIZONTAL_PADDING * (barsToDraw - 1)) / barsToDraw;
```

Comment:
why are you removing `BAR_HORIZONTAL_PADDING` in `ASSIGNEDBOX.w - BAR_HORIZONTAL_PADDING`?
it's just internal padding

Reply:
oh yeah..